CMakeLists.txt

```diff
@@ -120,9 +120,82 @@ foreach(tutorial_file ${tutorial_files})
 endforeach()
 add_custom_target(move_artifacts DEPENDS ${stamp_file} ${tutorial_files_builddir})
 
+add_subdirectory (interpreter)
```

Comment:
Why not moving this above this folder. On some platforms we enable modules for interpreter, too.